lib/pg_query/deparse.rb

```diff
@@ -377,6 +379,22 @@ def deparse_aexpr_any(node)
       output.join(' ' + deparse_item(node['name'][0], :operator) + ' ')
     end
 
+    def deparse_aexpr_between(node)
+      between = case node['kind']
+      when AEXPR_BETWEEN
```

Comment:
```
C: Indent when as deep as case.
      when AEXPR_BETWEEN
      ^^^^
```

Per our current style guidelines (enforced by rubocop) this would have to be indented as deep as the `case` keyword. Same goes for the other ones below.